epub33/core/index.html

```diff
@@ -7850,144 +7850,182 @@ <h4>Content document dimensions</h4>
+						aria-details="flow-paginated-multiple-diagram"
+						alt="The continuous progression of paginated content produced for each document with transitions to
+						new pages between documents." />
+				</figure>
+
+				<details id="flow-paginated-multiple-diagram" class="desc">
+					<summary>Image description</summary>
+					<p>Three column-like rectangles linked left-to-middle and middle-to-right with respective
+						arrows, with a text flowing from one rectangle to the next one. The text is sectioned with
+						headers figuring 'Chapter 1', '2'. The section with 'Chapter 2' starts at the top of the
+						rightmost rectangle, leaving an empty space at the bottom of the middle rectangle. The
+						leftmost rectangle is enclosed in a schematic view of a tablet.</p>
+				</details>
+
+				<figure id="fig-fxl-flow-paginated">
+					<figcaption>Rendering of a fixed layout EPUB publication with multiple spine items, and with the <code>rendition:flow</code> set to <code>paginated</code>.</figcaption>
+					<img src="images/example_rendering_FXL_paginated_multiple_spine.svg" width="600" 
+					aria-details="flow-fxl-paginated"
+					alt="The continuous progression of paginated content with transitions to new pages between spine items." />
+				</figure>
+
+				<details id="flow-fxl-paginated" class="desc">
+					<summary>Image description</summary>
+					<p>Three rectangles linked left-to-middle and middle-to-right with respective arrows, representing three spine items or pages in an EPUB document. Each page is styled like a comic, with segemented boxes within the rectangle. Each rectangle fills the screen of the schematic view of a tablet.</p>
+				</details>
+
+				<figure id="fig-flow-scrolled-continuous">
+					<figcaption>Rendering of an EPUB publication with a single spine item, and with the
+							<code>rendition:flow</code> set to <code>scrolled-continuous</code>.</figcaption>
+					<img src="images/example_rendering_scrolled_continuous.svg" width="220"
+						aria-details="flow-scrolled-continuous-diagram"
+						alt="The progression of a continuous scroll of content extends vertically off the user's screen,
+						with new documents added to the bottom as encountered." />
+				</figure>
+
+				<details id="flow-scrolled-continuous-diagram" class="desc">
+					<summary>Image description</summary>
+					<p>A single, column-like strip (i.e., a rectangle without a bottom edge) with a text flowing
+						down the strip. The text is sectioned with headers figuring 'Chapter 1', '2'. The top part
+						of the strip is enclosed in a schematic view of a tablet.</p>
+				</details>
+
+				<figure id="fig-fxl-flow-scrolled-continuous">
+					<figcaption>Rendering of a fixed layout EPUB publication with multiple spine items, and with the <code>rendition:flow</code> set to <code>scrolled-continuous</code>.</figcaption>
+					<img src="images/example_rendering_FXL_scrolled_continuous.svg" width="220"
```

Comment:
```suggestion
					<img src="images/example_rendering_FXL_scrolled_continuous.svg" width="900"
```

To make the size of the figure consistent with the others.